NEXTCLOUD.cmake

```diff
@@ -34,6 +34,7 @@ option( BUILD_UPDATER "Build updater" OFF )
 
 option( WITH_PROVIDERS "Build with providers list" ON )
 
+option( ENFORCE_VIRTUAL_FILES_SYNC_FOLDER "Enforce use of virtual files sync folder when available" OFF )
 
```

Comment:
Nit: why the spaces? We don't use it elsewhere.

Reply:
just look at the line `35` I just copied the style from previous lines

Reply:
Yes, but I mean other files and statements.